Detections/SecurityEvent/PotentialKerberoast.yaml

```diff
@@ -51,7 +51,7 @@ query: |
   (
   WindowsEvent
   | where TimeGenerated >= ago(starttime)
-  | where EventID == 4769 and EventData has '0x17' and EventData has '0x40810000' and EventData has 'krbtgt'
+  | where EventID == 4769 and EventData has '0x17' and EventData has '0x40810000'
```

Comment:
We will not remove the matching here. The query is including this and then only excluding if it is the machine account.  The query is much more complex than a simple krbtgt match or not.  If there is a need to modify to a specific environment, then simply modify the local template, the primary template does not need this change.